src/lib/validationSchema.ts

```diff
@@ -116,7 +136,7 @@ export const refreshmentValidationSchema = z.object({
   width: z.number().multipleOf(0.01),
   price: z.number().multipleOf(0.01),
   quantity: z.number().multipleOf(0.01),
-  unitType: z.string().uuid(),
+  unitType: z.enum(["กล่อง", "แก้ว", "ขวด", "ชิ้น"]),
```

Comment:
If you're gonna check enum here, why not just use enum in the schema?

Reply:
ภาษาไทย